src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,32 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article className={classNames(
+      'TodoInfo',
+      { 'TodoInfo--completed': completed },
+    )}
+    >
```

Comment:
Your article might look better if it's properly formatted for easy reading of your code
```suggestion
    
    <article className={
    classNames('TodoInfo', {
       'TodoInfo--completed': completed,
    })
    }
    >
```